tailwind.config.js

```diff
@@ -24,10 +24,24 @@ module.exports = {
         screens: {
             md: '599px',
             lg: '1023px',
+            xlg: '1280px',
```

Comment:
Matching the breakpoints set in `_variables.scss`

Reply:
great idea to update the tailwind spacing